### PR TITLE
Feature/managehh naming fix

### LINKED ISSUE #2222: Manage Household page doesn't work for New household object
hub thread: https://powerofus.force.com/0D58000002tINz9

### LINKED ISSUE #2294: With Locker Service on, Manage Household page's drag/drop of contacts is broken
With Locker Service on in Summer '16 or Winter '17, the drag/drop behavior for contact cards is broken.  The card being dragged gets displayed in the wrong location, and the drop has already gone thru and reordered the cards.

This is a LockerService bug that was reported in GUS, and has been marked fixed in release 206 which is Spring '17, which we can't yet verify.

Gus bug: W-3273335 
https://gus.lightning.force.com/one/one.app#/sObject/a07B0000002DaZJIA0/view

Gus bug: W-3346489
https://gus.lightning.force.com/one/one.app#/sObject/a07B0000002DaZJIA0/view

### LINKED ISSUE #2357: Rearranging contact cards in Manage Household does not update naming
Per this [Hub thread](https://powerofus.force.com/0D580000033yZjS). 

Replicated in brand new org.

image

image

### PR DESCRIPTION
**Pull request by davidhabib**
# Critical Changes

# Changes

# Issues Closed
Fixed #2357 
Fixed #2294 
Fixed #2222

### CODE REVIEW

**Comment by davidhabib** [comment]
@njjc ready for review and merge.  ignore the red failure (it's for Jason's mrbelvedere testing, not circleci).

**Comment by davidhabib** [comment]
@njjc I've added two more unrelated fixes to this pull request because they are all in ManageHHUI.  The one for #2222 was accepted by Mary as a workable fix.  In order to support old users who are using Household objects, and might have a Household object New override, or use a Household button, we needed ManageHouseholdUI to support that scenario.  You can read details in the AA item, but the short answer is that supporting a new object, rather than a saved object, would have been a non-trivial redesign for the lightning component and so we save the new account up front, before handing it off to the LC.  Let me know if you have any questions.  thanks.

File: src/classes/HH_ManageHH_TEST.cls [hunk 1]
```diff
@@ -88,4 +88,37 @@ private class HH_ManageHH_TEST {
...
         Test.stopTest();
     }
+
+    /*********************************************************************************************************
+    * @description Tests opening the ManageHouseholdUI page on an new HH Object
+    */
+    public static testMethod void testNewHHObject() { 
+        
+        npo02__Household__c hh = new npo02__Household__c(Name = 'test_hh');
+        insert hh;
```
Comment (jjbennett) [inline src/classes/HH_ManageHH_TEST.cls]: @davidhabib just curious why the test is creating the test hh, it doesn't appear to be used anywhere?

Comment (davidhabib) [inline src/classes/HH_ManageHH_TEST.cls]: looks like a cut/paste error.  i'll remove those two lines.

**Pull request merged**